docs/tempo/website/configuration/_index.md

```diff
@@ -254,6 +254,18 @@ querier:
     # The default value of "" disables this feature.
     [search_external_endpoints: <list of strings> | default = <empty list>]
 
+    # If search_external_endpoints is set then the querier will primarily act as a proxy for whatever serverless backend
+    # you have configured. This setting allows the operator to have the querier prefer itself for a configurable
+    # number of queries. In the default case of 2 the querier will process up to 2 search requests before starting
```

Comment:
This is more of a question to check my understanding:  `2 search requests` sounds like the external endpoint won't be hit unless there are concurrent search apis being called, but I think it is actually 2 search request shards? i.e. part of a whole search from the frontend as divided up among blocks/pages.

Reply:
> I think it is actually 2 search request shards?

This is correct. I will clarify.